content/bn/releases/_index.md

```diff
@@ -0,0 +1,28 @@
+---
+linktitle: প্রকাশের ইতিহাস
+Title: প্রকাশনাসমূহ
+Type: docs
+---
+
+
+<!-- overview -->
+
+কুবারনেটস প্রজেক্ট সাম্প্রতিক তিনটি পর্যন্ত ({{< skew latestVersion >}}, {{< skew prevMinorVersion >}}, {{< skew oldestMinorVersion >}}) ছোট রিলিজের জন্য রিলিজ শাখা বজায় রাখে।  Kubernetes 1.19 এবং নতুন ভার্সন আনুমানিক 1 বছরের প্যাচ সমর্থন পায়। Kubernetes 1.18 এবং তার বেশি বয়সীরা প্রায় 9 মাস প্যাচ সমর্থন পেয়েছে। 
+
+Kubernetes সংস্করণ **x.y.z** হিসাবে প্রকাশ করা হয়,
+যেখানে **x** হল প্রধান সংস্করণ, **y** হল অপ্রধান সংস্করণ এবং **z** হল প্যাচ সংস্করণ, যা [শব্দার্থিক সংস্করণ](https://semver.org/) পরিভাষা অনুসরণ করে হয়। 
+
+অতিরিক্ত তথ্যসমূহ [version skew policy](/releases/version-skew-policy/) নথিতে সংরক্ষিত রয়েছে। 
+
+
+<!-- body -->
+
+## প্রকাশের ইতিহাস
+
+{{< release-data >}}
+
+## আসন্ন রিলিজ
+
+চেক করুন [সময়সূচী](https://github.com/kubernetes/sig-release/tree/master/releases/release-{{< skew nextMinorVersion >}}) আসন্ন **{{< skew nextMinorVersion >}}** Kubernetes মুক্তিপ্রকাশ!
```

Comment:
Can you check this sentence?

Reply:
Yes obviously! I have tried to fix it. Can you take a look? @mitul3737

Reply:
Checked and approved